.travis.yml

```diff
@@ -7,7 +7,7 @@ cache:
     - $HOME/.composer/cache/files
 
 php:
-  - 7
+  - 7.1
```

Comment:
It is stable enough ('been using it for a while), I just need to make sure that it is installed in the deployment environment (ubuntu box)